cape-char.el

```diff
@@ -25,44 +25,35 @@
 ;;; Code:
 
 (require 'cape)
+(require 'quail)
 
 (autoload 'thing-at-point-looking-at "thingatpt")
 
 ;; Declare as pure function which is evaluated at compile time. We don't use a
 ;; macro for this computation since packages like `helpful' will
 ;; `macroexpand-all' the expensive `cape-char--define' macro calls.
 (eval-when-compile
-  (defun cape-char--translation (method regexp)
```

Comment:
The regexp argument should be kept to filter the char names, since otherwise the resulting hash table will contain keys which do not match the prefix. There is no visible effect because of the completion filtering but we will save a little memory.

Reply:
[8a58051](https://github.com/minad/cape/pull/91/commits/8a5805143e5b84855b7be9b196224e657c8b90b9)